### PR TITLE
LTD-915: Add browser test to Review and approve an Organisation

### PR DESCRIPTION
**Pull request by saruniitr**
## Change description

Once an organisation is registered then Caseworker need to review
and approve/reject it. This change adds a browser test to ensure that
the organisation is reviewed and approved.

### CODE REVIEW

File: tests_common/api_client/libraries/request_data.py [hunk 1]
```diff
@@ -50,13 +50,17 @@ def build_organisation_with_user(exporter, type, name):
 
 
 def build_organisation(name, type, address):
+    eori_number = "GB" + "".join([str(random.randint(0, 9)) for _ in range(12)])
```
Comment (alixedi) [inline tests_common/api_client/libraries/request_data.py:53]: Might want to move this to a function and refactor [this](https://github.com/uktrade/lite-frontend/pull/204/files#diff-de04886c76fb2582ef9b97d55ab2b3feff7fdf8b7e27f1fa64c26b741390764fR55) accordingly

File: tests_common/api_client/libraries/request_data.py [hunk 2]
```diff
@@ -50,13 +50,17 @@ def build_organisation_with_user(exporter, type, name):
...
+    registration_number = "".join([str(random.randint(0, 9)) for _ in range(8)])
```
Comment (alixedi) [inline tests_common/api_client/libraries/request_data.py:54]: Same here.

File: ui_tests/caseworker/features/organisation.feature [hunk 3]
```diff
@@ -34,27 +34,21 @@ Feature: I want to add a company to LITE
     When I click the organisation
     Then the "created" organisation appears in the audit trail
 
-  @skip @LT_1105_review_and_approve_an_organisation @regression
+  @LTD-915-Review-approve-organisation
```
Comment (alixedi) [inline ui_tests/caseworker/features/organisation.feature:37]: Is there a reason to mark the scenarios with the ticket number? (given that JIRA will link your PR to the ticket as well)

**Review by alixedi — APPROVED** [review]
A few comments that I will leave with you.

File: tests_common/api_client/libraries/request_data.py (hunk 1 above)
Comment (saruniitr) [reply tests_common/api_client/libraries/request_data.py:53]: :+1:

File: ui_tests/caseworker/features/organisation.feature (hunk 3 above)
Comment (saruniitr) [reply ui_tests/caseworker/features/organisation.feature:37]: It is useful when running the test locally, you can use these tags to run a specific test.

**Pull request merged**